core/contracts/financial_templates/Liquidatable.sol

```diff
@@ -91,9 +61,31 @@ contract Liquidatable is PricelessPositionManager {
     // Represented as a multipler, see above
     FixedPoint.Unsigned disputerDisputeRewardPct;
 
-    /**
-     * Method modifiers
-     */
+    event LiquidationCreated(
+        address sponsor,
+        address liquidator,
+        uint liquidationId,
+        uint tokensOutstanding,
+        uint lockedCollateral,
+        uint liquidatedCollateral
+    );
+    event LiquidationDisputed(
+        address sponsor,
+        address liquidator,
+        address disputer,
+        uint disputeId,
+        uint disputeBondAmount
+    );
+    event DisputeSettled(
+        address caller,
+        address sponsor,
+        address liquidator,
+        address disputer,
+        uint disputeId,
+        bool DisputeSucceeded
+    );
```

Comment:
Can/should some of these params be indexed?

Reply:
We can for sure index some. Will refactor and add logical indices.